cmd/cli/util.go

```diff
@@ -113,25 +114,21 @@ func getMeshInfoList(restConfig *rest.Config, clientSet kubernetes.Interface) ([
 	return meshInfoList, nil
 }
 
-// getNamespacePods returns a map of controller pods
-func getNamespacePods(clientSet kubernetes.Interface, m string, ns string) map[string][]string {
-	x := make(map[string][]string)
-
+// GetNamespaceOsmControllerPods returns a list of osm-controller pods in the namespace
+func GetNamespaceOsmControllerPods(clientSet kubernetes.Interface, ns string) *corev1.PodList {
 	labelSelector := metav1.LabelSelector{MatchLabels: map[string]string{"app": constants.OSMControllerName}}
 	listOptions := metav1.ListOptions{
 		LabelSelector: labels.Set(labelSelector.MatchLabels).String(),
 	}
-	pods, _ := clientSet.CoreV1().Pods(ns).List(context.TODO(), listOptions)
-
-	for pno := 0; pno < len(pods.Items); pno++ {
-		x["Pods"] = append(x["Pods"], pods.Items[pno].GetName())
+	podList, err := clientSet.CoreV1().Pods(ns).List(context.TODO(), listOptions)
+	if err != nil {
+		return &corev1.PodList{}
```

Comment:
Drop L127 and:
```suggestion
	podList, _:= clientSet.CoreV1().Pods(ns).List(context.TODO(), listOptions)
	return podList
```

The API should return a nil return on error. Alternatively, return an error from this function and let the caller handle it.